src/utils/event.ts

```diff
@@ -57,7 +60,24 @@ export function warpEventCanonicalSignaturehash(eventName: string, argTypes: arg
   const funcSignature = `${eventName}(${argTypes.map(getArgStringRepresentation).join(',')})`;
   const funcSignatureHash = createKeccakHash('keccak256').update(funcSignature).digest('hex');
 
-  return `0x${(
-    BigInt(`0x${createKeccakHash('keccak256').update(funcSignatureHash).digest('hex')}`) & MASK_250
-  ).toString(16)}`;
+  const [low, high] = toUintOrFelt(BigInt(`0x${funcSignatureHash}`), 256);
+
+  return {
+    low: `0x${low.toString(16)}`,
+    high: `0x${high.toString(16)}`,
+  };
+}
```

Comment:
Couldn't the function call `warpEvenSignatureHash256FromString`? seems that the logic since `const funcSignatureHash`  is the same